instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy.rb

```diff
@@ -4,6 +4,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+require 'base64'
```

Comment:
Should this be a dependency of trilogy or the vitess prop?

Reply:
Vitess - thx.